repose-aggregator/tests/release-verification/build.gradle

```diff
@@ -0,0 +1,250 @@
+// It also forces the usage of jackson dependencies which were not being brought in as expected
+// TODO: Remove unix-socket-factory after upgrading to gradle-docker-plugin 3.0.4
+configurations {
+    dockerJava {
+        resolutionStrategy {
+            force 'de.gesellix:unix-socket-factory:2016-04-06T22-21-19'
+            force 'com.fasterxml.jackson.core:jackson-databind:2.6.4'
+            force 'com.fasterxml.jackson.core:jackson-core:2.6.4'
+            force 'com.fasterxml.jackson.core:jackson-annotations:2.6.4'
+        }
+    }
+}
+
+docker {
+    url = "unix:///var/run/docker.sock"
+}
+
+task buildDebImage(type: DockerBuildImage) {
+    dependsOn copyDebFiles
+    inputDir = file("$buildDir/deb")
+    tag = 'repose:deb-release-verification'
+    buildArgs = ['REPOSE_VERSION': releaseVersion]
+}
+
+task buildRpmImage(type: DockerBuildImage) {
+    dependsOn copyRpmFiles
+    inputDir = file("$buildDir/rpm")
+    tag = 'repose:rpm-release-verification'
+    buildArgs = ['REPOSE_VERSION': releaseVersion]
+}
+
+task createDebContainer(type: DockerCreateContainer) {
+    dependsOn buildDebImage
+    targetImageId { buildDebImage.getImageId() }
+    portBindings = ['10037:18038', '8080:18088']
+}
+
+task createRpmContainer(type: DockerCreateContainer) {
+    dependsOn buildRpmImage
+    targetImageId { buildRpmImage.getImageId() }
+    portBindings = ['10037:18038', '8080:18088']
+}
+
+task startDebContainer(type: DockerStartContainer) {
+    dependsOn createDebContainer
+    targetContainerId { createDebContainer.getContainerId() }
+}
+
+task startRpmContainer(type: DockerStartContainer) {
+    dependsOn createRpmContainer
+    targetContainerId { createRpmContainer.getContainerId() }
+}
+
+task stopDebContainer(type: DockerStopContainer) {
+    dependsOn startDebContainer
+    targetContainerId { startDebContainer.getContainerId() }
+}
+
+task stopRpmContainer(type: DockerStopContainer) {
+    dependsOn startRpmContainer
+    targetContainerId { startRpmContainer.getContainerId() }
+}
+
+task removeDebContainer(type: DockerRemoveContainer) {
+    dependsOn stopDebContainer
+    targetContainerId { stopDebContainer.getContainerId() }
+}
+
+task removeRpmContainer(type: DockerRemoveContainer) {
+    dependsOn stopRpmContainer
+    targetContainerId { stopRpmContainer.getContainerId() }
+}
+
+task removeDebImage(type: DockerRemoveImage) {
+    dependsOn buildDebImage
+    targetImageId { buildDebImage.getImageId() }
+}
+
+task removeRpmImage(type: DockerRemoveImage) {
+    dependsOn buildRpmImage
+    targetImageId { buildRpmImage.getImageId() }
+}
+
+task dockerSmokeTest {
+    description = 'tests if the Repose instance can handle basic requests using Docker'
+    group = 'release'
+    dependsOn = ['buildDebImage',
+                 'buildRpmImage']
+    finalizedBy = ['removeDebImage',
+                   'removeRpmImage']
+}
+
+// Note: At this time, release verification will be handled out-of-band with the release. Therefore,
```

Comment:
Fair enough. I think I convinced myself that finalizedBy tasks would run if the task was scheduled to run as opposed to if the task ran. That is not the case, though. I'll add in the other tasks and finalize them.